build.cake

```diff
@@ -5,14 +5,22 @@
 // ARGUMENTS
 //////////////////////////////////////////////////////////////////////
 
+const string DefaultVersion = "1.0.0.0";
+
 var target = Argument("build-target", "Default");
-var version = Argument("build-version", "1.0.0.0");
+var version = Argument("build-version", DefaultVersion);
 var configuration = Argument("build-configuration", "Release");
 
+if (version == DefaultVersion)
+    Warning($"No version specified. Using default version '{DefaultVersion}'.");
+
 //////////////////////////////////////////////////////////////////////
 // PREPARATION
 //////////////////////////////////////////////////////////////////////
 
+var majorVersion = "3.0";
+var buildNumber = EnvironmentVariable("BUILD_NUMBER");
```

Comment:
Add a way to specify these using arguments.

Reply:
I just copied how they were set in the old build script.
I think we should wait with fixing this until we set up proper versioning (I want to set up NerdBank.GitVersioning).